backend/shared/middleware/auth.ts

```
@@ -0,0 +1,51 @@
+ * Middleware to authenticate JWT token
+ * @param req Express request
+ * @param res Express response
+ * @param next Express next function
+ */
+export const authenticateToken = (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  const authHeader = req.headers['authorization'];
+  const token = authHeader && authHeader.split(' ')[1];  // Bearer TOKEN
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)